src/app/+dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,20 @@
+@import '../core/styles/colors';
+
+mat-grid-list {
+    margin: 10px;
+}
+
+mat-grid-tile {
+    background-color: $grey;
+    border-radius: 5px;
+}
+
+.greeting {
+    background-color: $green;
+}
+
+.w-h-100 {
+    width: 100%;
+    height: 100%;
+    color: #fff;
```

Comment:
if you provide such classes (w-100, h-50, etc) try to keep only described in the name styles. it is not obvious that you will get white color here. Separate this style or rename a class, please.

Reply:
fixed